src/tls13.c

```diff
@@ -6073,6 +6073,9 @@ static int SendTls13CertificateVerify(WOLFSSL* ssl)
 
         case TLS_ASYNC_BUILD:
         {
+            word32 rem = ssl->buffers.outputBuffer.bufferSize
```

Comment:
Perhaps consider using signed int for this calculation to better handle the case to prevent possible negative here? Possibly re-write as:

```
int rem = (int)ssl->buffers.outputBuffer.bufferSize - (int)ssl->buffers.outputBuffer.length - RECORD_HEADER_SZ - HANDSHAKE_HEADER_SZ;

if (rem < 0 || args->length > rem) {
    ERROR_OUT(MP_EXPTMOD_E, exit_scv);
}
```

Reply:
Also - keep lines to 80 characters.